packages/block-library/src/archives/editor.scss

```diff
@@ -0,0 +1,3 @@
+.gutenberg ul.wp-block-archives {
```

Comment:
Is there a specific reason why it is `.gutenberg ul.wp-block-archives` and not just `.gutenberg .wp-block-archives`?

Reply:
Yes, there is. The Archives block can be displayed as dropdown and the class '.wp-block-archives' is used for div element.